tracer/build/_build/Build.Steps.cs

```diff
@@ -2043,9 +2055,19 @@ private void IntegrationTestLinuxProfilerDirFudge(string project)
         CopyDirectoryRecursively(src, dest, DirectoryExistsPolicy.Merge, FileExistsPolicy.Overwrite);
 
         // not sure exactly where this is supposed to go, may need to change the original build
-        foreach (var linuxDir in MonitoringHomeDirectory.GlobDirectories("linux-*"))
+        if (IsLinux)
         {
-            CopyDirectoryRecursively(linuxDir, dest, DirectoryExistsPolicy.Merge, FileExistsPolicy.Overwrite);
+            foreach (var linuxDir in MonitoringHomeDirectory.GlobDirectories("linux-*"))
+            {
+                CopyDirectoryRecursively(linuxDir, dest, DirectoryExistsPolicy.Merge, FileExistsPolicy.Overwrite);
+            }
+        }
+        else if (IsOsx)
+        {
+            foreach (var linuxDir in MonitoringHomeDirectory.GlobDirectories("osx"))
```

Comment:
Feel free to ignore obv
```suggestion
            foreach (var osxDir in MonitoringHomeDirectory.GlobDirectories("osx"))
```

Reply:
done in [7f42b05](https://github.com/DataDog/dd-trace-dotnet/pull/3863/commits/7f42b05e794ac84b30e5fc059d4d6c8c7b8eb4af)